### ISSUE TITLE
Experimentation: non technical homepage

### ISSUE THREAD

**Issue by Mogztter** [asciidoctor/asciidoctor.org#646]
### Current situation

The current homepage introduces a lot of technical details. If you are a technical person, this is good because, you can start right away and download the `asciidoctor` Gem or checkout the source from GitHub.

On the contrary, if you are a non technical person, you will most likely move on... It's a pity, because we now have "low barriers" tools that are using Asciidoctor. For instance, I think a non technical person can install the Atom editor and the AsciiDoc preview package without too much trouble. Or write some AsciiDoc in Microsoft Notepad 🙈 and then use the Chrome extension.

### Proposal

![asciidoctor-home4](https://cloud.githubusercontent.com/assets/333276/21924622/3d40efba-d97a-11e6-8aa8-ffa83f48cac2.png)

I've removed all the technical details except the source code block in the AsciiDoc syntax example.
The structure of the page is:
* Syntax (AsciiDoc)
* Processor
* Integration

Basically:

1. Syntax = Write
2. Processor = Publish
3. Integration = How do I setup Asciidoctor in the tools that I'm already using / Getting started

Let me know what you think 😉

**Comment by Mogztter** [comment]
I think we should also use an illustration to explain what is Asciidoctor, something like:

![asciidoctor-nutshell](https://cloud.githubusercontent.com/assets/333276/21925556/4de84526-d97e-11e6-9a79-0777d927626b.png)

AsciiDoc syntax comes in, HTML5 comes out, easy!
Credit to [OpenDevise](http://opendevise.com/) for the adoc document in the illustration 👍

**Comment by Mogztter** [comment]
Why not :)

I think we could use http://gist.asciidoctor.org for the "try-it" button
even though this application will ask for a GitHub account (but this is not
mandatory).

**Comment by rockyallen** [comment]
This would be good opportunity to improve navigation and make it feel more like an integrated website:
1. Keep the top bar (the one in black) fixed at the top of the screen in all pages and  documents.
2. Turn "Asciidoctor" on the left into a crumb trail:

Asciidoctor (=home page)
Asciidoctor - News
Asciidoctor - docs
Asciidoctor - docs - Installation
Asciidoctor - docs - User guide
Asciidoctor - docs - Migration guide
etc

although I realize this may be limited by what JBake provides.
